cli/cmd/generate.go

```diff
@@ -96,21 +99,25 @@ func runGenerate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	paths, cmPaths, err := findGenerateTargets(args, log)
+	paths, err := findYamlFiles(args)
 	if err != nil {
 		return err
 	}
-	if flags.outputFile != "" {
-		tmpDir, newPaths, err := getTmpPaths(paths)
-		if err != nil {
-			return fmt.Errorf("get temporary paths: %w", err)
-		}
-		defer os.RemoveAll(tmpDir)
-		paths = newPaths
+
+	extraFile, err := os.CreateTemp("", "contrast-generate-extra-*.yml")
+	if err != nil {
+		return fmt.Errorf("create temp file for configmaps/secrets: %w", err)
+	}
+	defer extraFile.Close()
```

Comment:
Doesn't this need to be closed after `extractTargets`, if only to force flushing?